pycloudlib/ec2/instance.py

```diff
@@ -15,6 +15,7 @@ class EC2Instance(BaseInstance):
     """EC2 backed instance."""
 
     _type = "ec2"
+    ready_timeout = 40  # To keep backwards compatibility (will lower soon)
```

Comment:
```suggestion
    ready_timeout = 40  # To keep backwards compatibility (will lower soon™)
```
(this is a joke, don't actually apply this)